releases/release-1.31/release-notes/maps/pr-124465-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 124465
+releasenote:
+  text: 'kubeadm: switch to using the new etcd endpoints introduced in 3.5.11 - /livez
+    (for liveness probe) and /readyz (for readyness and startup probe). With this
+    change it is no longer possible to deploy a custom etcd version older than 3.5.11
+    with kubeadm 1.31. If so, please upgrade.'
```

Comment:
Not sure what is being said here "If so, please upgrade". Are they saying if they require to deploy a custom etcd version OLDER THAN 3.5.11, they would need to upgrade Kubeadm? or dont use custom etcd version older than 3.5.11 and they should upgrade etcd?

Reply:
I think they're hoping this encourages people to upgrade etcd: https://github.com/kubernetes/kubeadm/issues/3039 If you look at the PR the supported etcd versions have updated, so in 1.31 users will need to make sure they're on a supported etcd version. Maybe we can reword this slightly to make it more clear? `If so, please upgrade etcd to a supported version.`

Reply:
Fixed